domains/certificates/Issuer/Issuer.Worker/Program.cs

```diff
@@ -5,35 +5,37 @@
 using Issuer.Worker.MasterDataService;
 using Issuer.Worker.QueryModelUpdater;
 using Issuer.Worker.RegistryConnector;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Serilog;
 using Serilog.Formatting.Json;
 
-var host = Host.CreateDefaultBuilder(args)
-    .ConfigureLogging((context, builder) =>
-    {
-        builder.ClearProviders();
+var builder = WebApplication.CreateBuilder(args);
 
-        var loggerConfiguration = new LoggerConfiguration();
+var loggerConfiguration = new LoggerConfiguration();
+var logger = builder.Environment.IsDevelopment()
+    ? loggerConfiguration.WriteTo.Console()
+    : loggerConfiguration.WriteTo.Console(new JsonFormatter());
 
-        loggerConfiguration = context.HostingEnvironment.IsDevelopment()
-            ? loggerConfiguration.WriteTo.Console()
-            : loggerConfiguration.WriteTo.Console(new JsonFormatter());
+builder.Logging.ClearProviders();
+builder.Logging.AddSerilog(logger
+    .Filter
+    .ByExcluding("RequestPath like '/health%'")
```

Comment:
I think this should be moved, so it is added to `loggerConfiguration`. By doing that we have all serilog configuration close together.